Scrapper-Service/README.md

```diff
@@ -38,6 +38,30 @@ optional arguments:
 (base)
 ```
 
+## Deploying Scraper service in different modes
+
+```bash
+# running the scraper with a custom configuration file
+>> python app.py -c <path/to/config.json>
+or
+>> python app.py --config <path/to/config.json>
+
+# running the scraper service with logs of specific type
+>> python app.py -l debug # for only debug logs
+>> python app.py -l warn # for only warn logs
+>> python app.py -l error # for only error logs
+>> python app.py -l info # for only info logs
+
+# running the scraper service with different test modes
+>> python app.py --t False
```

Comment:
* Do mention what test mode False does i.e runs the application and spawns the threads for starting the processes. 
* Same in true , but does not spawns threads and runs sequentially to test connection and basic setup.